testgrid/specs/deploy.yaml

```diff
@@ -376,3 +376,34 @@
     validate_read_write_object_store postupgrade upgradefile.txt
   unsupportedOSIDs:
   - rocky-91 # docker is not supported on rhel 9 variants
+- name: latest install with all metrics and monitor addons enabled with kots
```

Comment:
did you test that this spec will not run out of cpu? also could you just add collectd to an existing spec rather than add a new one?

Reply:
I am testing the specs
We have no test to check ALL metrics addons enabled
So, I think we should add it on the daily one.

Reply:
I am closing this one since changes were applied and the description were updated